README.md

```
@@ -32,33 +32,33 @@ Table end
     $ git clone --recursive https://github.com/sclorg/nginx-container.git
     $ cd nginx-container
     $ git submodule update --init
-    $ make build TARGET=rhel9 VERSIONS=1.24
+    $ make build TARGET=rhel10 VERSIONS=1.26
     ```
 
 *  **CentOS Stream based image**
 
     This image is available on DockerHub. To download it run:
 
     ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)